flytekit/core/type_engine.py

```diff
@@ -221,6 +224,13 @@ def get_literal_type(self, t: Type[T]) -> LiteralType:
         Extracts the Literal type definition for a Dataclass and returns a type Struct.
         If possible also extracts the JSONSchema for the dataclass.
         """
+        if get_origin(t) is typing.Annotated:
+            raise ValueError(
+                f"Flytekit does not currently have support \
+                    for FlyteAnnotations applied to Dataclass. {t} cannot be \
+                    parsed."
+            )
+
```

Comment:
Split this multi-line string in multiple strings instead, e.g.:
```
raise ValueError(
    "Flytekit does not currently have support for FlyteAnnotations applied to Dataclass. "
    f"Type {t} cannot be parsed."
)
```

This will allow for people to grep / skip these log lines more easily.

That said, we should probably be more prescriptive here (i.e. add a max column number in our linters). Not in this PR obviously.

Reply:
made the change to avoid another iteration, but can you describe why making the log multiple strings makes it more "greppable"?

Reply:
Sure, what I mean is that it'll be easier to come up with a regex if you don't have to worry about counting spaces.

For example, as it was before, the string present in the `ValueError` would show up in the logs as:
```
'Flytekit does not currently have support             for FlyteAnnotations applied to Dataclass. Annotated[SomeDataClass, FlyteAnnotation()] cannot be             parsed.
``` 

Notice how one have to worry about coming up with a regex that takes into account the extra spaces, as opposed to the more straightforward:
```
'Flytekit does not currently have support for FlyteAnnotations applied to Dataclass. Annotated[SomeDataClass, FlyteAnnotation()] cannot be parsed.
```